libraries/lib-module-manager/PluginDescriptor.h

```diff
@@ -12,6 +12,8 @@
 #include <wx/string.h>
 
 #include "PluginInterface.h"
+#include "XMLTagHandler.h"
+#include "XMLWriter.h"
```

Comment:
This header only needs a forward declaration of type `XMLWriter` and should avoid the nested header